benches/collatz.rs

```diff
@@ -6,15 +6,12 @@ crepe! {
     @input
     struct Start(u128);
```

Comment:
While we're here can you indent this to match the above '@input'

Reply:
Happy to adjust the indentation, but how exactly? This is the same indentation as in the readme and in the other benchmark, with 4 spaces both before `@input` and `struct` :thinking:.